client/state/signup/dependency-store/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getSignupDependencyStore( state ) {
+	if ( state && state.signup && state.signup.dependencyStore ) {
```

Comment:
`get` is handy for circumstances where it's possible that one of the path fragments will be `undefined`, but when will that ever be the case here? You can (or *should*) always be confident that a reducer path exists. Rather, I don't think you need this condition or `get` at all:

```js
return state.signup.dependencyStore;
```